charts/adot-exporter-for-eks-on-ec2/templates/adot-collector/configmap.yaml

```diff
@@ -42,14 +44,13 @@ data:
         - {{.}}{{- end }}
         metric_declarations:
           {{ .Values.adotCollector.daemonSet.metricDeclarations | nindent 10 }}
-      awsprometheusremotewrite:
+      prometheusremotewrite:
         namespace: {{ .Values.adotCollector.daemonSet.ampexporters.namespace }}
         endpoint: {{ .Values.adotCollector.daemonSet.ampexporters.endpoint }}
         resource_to_telemetry_conversion: 
-          enabled: {{ .Values.adotCollector.daemonSet.ampexporters.resourcetotel }}
-        aws_auth:
-          region: {{ .Values.adotCollector.daemonSet.ampexporters.region }}
-          service: aps
+          enabled: {{ .Values.adotCollector.daemonSet.ampexporters.resourcetootel }}
+        auth:
+          authenticator: {{ .Values.adotCollector.daemonSet.extensions.sigv4auth }}
```

Comment:
is `sigv4auth` a value in values.yaml? In it, it looks like this,
```
extensions:
  healthCheck: ""
  sigv4auth:
    region: ""
```
will `{{ .Values.adotCollector.daemonSet.extensions.sigv4auth }}` return `sigv4auth` as a value?